CHANGELOG.md

```diff
@@ -8,6 +9,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changes
 
 - Upgraded to timely dataflow and differential dataflow dependencies to v0.12.
+- Worked on improving the debuggability of ddlog dataflow graphs
+
+### Bug Fixes
+
+- Fixed crashes due to misused unsafe code
```

Comment:
I don't think this needs to be in the changelog. The code only became buggy when nested scoped were introduced, so there was never a bug in the master branch; it only transiently existed in this PR.